web-ui/src/main/resources/catalog/components/viewer/wps/WpsDirective.js

```diff
@@ -57,44 +45,69 @@
           scope.status = 'loading';
           gnWpsService.describeProcess(scope.uri, scope.processId)
```

Comment:
i would add `var defaults = scope.defaults || {};` and use this variable later in the code